<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: rgb(13, 16, 17);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Fecha Nacimiento</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Apellido</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Edad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Cod. Cliente</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Teléfono</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DNI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Sexo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>310</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Dirección</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Masculino</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Femenino</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>29</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>31</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>33</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>34</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>35</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>36</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>37</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>39</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>40</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>41</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>42</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>43</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>44</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>45</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>46</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>47</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>48</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>49</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>51</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>52</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>53</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>54</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>55</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>56</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>58</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>59</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>61</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>62</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>63</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>64</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>65</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>66</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>67</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>68</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>69</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>70</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>80</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>81</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>82</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>83</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>84</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>85</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>86</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>87</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>88</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>89</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>90</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>91</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>92</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>93</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>94</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>95</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>96</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>97</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>98</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>99</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1011</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>dripicons-v2</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27, 42, 54);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Icons/chevrons-left.svg</normaloff>Icons/chevrons-left.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>961</width>
      <height>261</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Teléfono</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Edad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DNI</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha Nacimiento</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dirección</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sexo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cod. Cliente</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Mantenimiento Clientes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Listar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Borrar</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
